tests/test_client.py

```diff
@@ -42,12 +42,10 @@ def test_client_request_auth(token):
 
 @pytest.mark.vcr()
 async def test_async_client_request_auth(token):
-    async_client = AsyncClient()
+    async_client = AsyncClient(auth="Invalid")
 
     with pytest.raises(APIResponseError):
         await async_client.request("/users", "GET")
 
     response = await async_client.request("/users", "GET", auth=token)
     assert response["results"]
-
-    await async_client.aclose()
```

Comment:
I removed aclose because I didn't want to have a closed client to interfere with the other tests, and the line was already covered in conftest (async_client). Probably it is unnecessary but I like to keep both sync and async tests the same, and if I close the sync client the other tests were failing (being a module fixture). 
Let me know if the explanation makes sense aha :)

Reply:
Mhh, in this test and the one just above, client and async_client aren't fixture, they are instantiated within the tests. That's why we need to close them, otherwise they're not going to be closed ever and that could give us some warning at the ends of the tests.

Reply:
All clear now! Fixed it